src/main/scala-2/chisel3/aop/Select.scala

```diff
@@ -51,6 +52,177 @@ object Select {
     }
   }
 
+  def dedupHash[T <: BaseModule](hier: Definition[T]): Int = {
```

Comment:
```suggestion
  def unsafeDedupHash[T <: BaseModule](hier: Definition[T]): Int = {
```

or `estimatedDedupHash` or something equally "don't trust this"-y?